### ISSUE TITLE
Show full path

### ISSUE THREAD

**Issue by melvin4u445** [kirillt/ark-navigator#56]
Another improvement. It would be great if the user had the option to see the full path to the opened folder. There isn't enough space to display the full path on the header, so maybe we could show it somewhere in the white section? The header could just show the name of the current folder or something?

![image](https://user-images.githubusercontent.com/17034768/132106424-2f38dd7c-1f6e-4559-9d41-810df76adf54.png)

**Comment by kirillt** [comment]
That's right.. Honestly I don't know how to do it in terms of UX, but yes we need this. Maybe just header in white section which would be not visible when a user scrolls down, but the user always can bring it back by scrolling up?

**Comment by melvin4u445** [comment]
Yeah, that suggestion sounds good I can think of at least one more way to display this. An `i` (info) button on the header bar, which opens a dialog/view with more details like path, number of files, tags, etc. We could use this section in the future to add more relevant information. In my opinion, a view (I guess that's what it's called?) would be better than a dialog.

**Comment by kirillt** [comment]
That's good idea to create pop-up widget with information. Can be done with `i` button, or we can just make the header clickable.
